src/specs/models/processing-config-bodies/ProcessingConfigConfigureEmbedding.v1.yaml

```diff
@@ -0,0 +1,51 @@
+title: Configure Embedding
+type: object
+properties:
+  embeddingSettings:
+    type: object
+    required:
+      - method
+      - methodSettings
+    properties:
+      method:
+        type: string
+        minLength: 1
+      methodSettings:
+        type: object
+        properties:
+          umap:
+            type: object
+            properties:
+              minimumDistance:
+                type: number
+              distanceMetric:
+                type: string
+                minLength: 1
+          tsne:
```

Comment:
this is essentially the same config that we use for the `GetEmbedding` task -- writing it again in here means that if we want to make any changes to the schema, we will have to update it both here and in `GetEmbedding`, which is not ideal.

Reply:
Correct, we are not currently checking the embedding config to comply with any schema. If we were to do this, I would do it in a separate ticket as this is out of scope for this change.